### PR TITLE
Add BNB spaceid view registrations

### PR DESCRIPTION
**Pull request by springzh**
Brief comments on the purpose of your changes:

This is a simple spell that merge all bnb space id name registration event together into a view. We plan to use it as a sample for our tutorial course for Chinese users who are studing Dune analytics. So please review asap.  Thank you.

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/spaceid/bnb/spaceid_bnb_schema.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        name: version
+        description: "Contract version"
+      - &block_time
+        name: block_time
+        description: "UTC event block time"
+      - &name
+        name: name
+        description: "Name of the space ID"
+        tests:
+          - unique
```
Comment (Hosuke) [inline models/spaceid/bnb/spaceid_bnb_schema.yml:24]: The unique test for `name` column failed.
https://dune.com/queries/2019807?d=1

Comment (Hosuke) [reply models/spaceid/bnb/spaceid_bnb_schema.yml:24]: The dupes are from V5 and V9
https://dune.com/queries/2019828

File: models/spaceid/bnb/spaceid_bnb_schema.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+version: 2
+
+models:
+  - name: spaceid_bnb_view_registrations
```
Comment (Hosuke) [inline models/spaceid/bnb/spaceid_bnb_schema.yml:4]: We can simply name the model as `spaceid_bnb_registrations`, also we need to change the alias.

Comment (springzh) [reply models/spaceid/bnb/spaceid_bnb_schema.yml]: Okay. Renamed.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @springzh 

[Sample bnb sapceid registrations](https://dune.com/queries/2019970)

**Review by jeff-dude — APPROVED** [review]
thank you @springzh and apologies for the delay for review/merge.

**Comment by jeff-dude** [comment]
fyi, i removed one of two unique tests.

**Pull request merged**